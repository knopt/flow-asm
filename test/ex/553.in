29
1
-1.512816806017053
5.307380956273917 8.293948975809506 0.7456576183983099 -2.9299028238300417 -1.2509413438023298 3.199123673967815 4.366757899217166 8.629872373712303 -9.29049161272619 -1.0573586263924621 -5.61143293405528 -2.3602078909410107 -0.531370431326657 -4.140744032627954 -8.641544834236079 -9.169840458103145 3.8014198602298954 9.979442261251585 8.531599270844335 0.9693334226476047 -2.999491216565822 2.8387256521390327 5.369205041243646 -8.421140235819172 -2.0102018455711113 6.555552683628889 -6.371441764046734 7.396833605819641 8.494777496338518 

-8.887584190146892 
