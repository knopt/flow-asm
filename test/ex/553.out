26.781762790605637 12.812079268076012 -10.673524403806622 -8.490352432364563 1.2890197998388575 9.931256820719067 6.133174578455086 15.079183596703103 -36.40071942135309 11.397863321506568 -12.500913082538066 2.5582999944257736 2.235325613051471 -9.601065075870851 -15.450431927443875 -9.969054956434507 23.42456046502623 19.325658377467022 6.341278062482262 -10.470989444073213 -9.003595830902462 11.670878248287991 9.197356788360871 -29.283406331737694 7.688373493536067 19.51397009161934 -25.92761621577955 28.22571197522415 10.155765465979222 
